manifests/master/routes.pp

```diff
@@ -22,7 +22,7 @@
   } elsif $routes {
     $routes_real = $routes
   } else {
-    if versioncmp($serverversion, '7.0') == -1 {
+    if (defined('$serverversion')) and (versioncmp($serverversion, '7.0') == 1) {
```

Comment:
I think we still want `-1` here so we use yaml on < 7.0:

```
$ bx puppet apply -e "notice(versioncmp('6.19', '7.0'))"
Notice: Scope(Class[main]): -1
```

Reply:
Updated.
Now if serverversion is defined and if the version is greater than 7.0, we're going to prefer json. If not we're going to prefer yaml